common/app/views/fragments/immersiveGalleryHeadTonal.scala.html

```diff
@@ -0,0 +1,90 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive content__head--desktop is-fixed-height tonal__head tonal__head--@toneClass(item)">
```

Comment:
I will take a look if it is worth moving headerImmersive to `common/`. It is a good point that it may not be necessary to use both.